src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/elasticapi/ElasticExtensions.kt

```diff
@@ -210,6 +215,7 @@ class InjectorContextElement(
 
     override fun updateThreadContext(context: CoroutineContext) {
         rolesInjectorHelper.injectRoles(roles)
+        if (user != null) rolesInjectorHelper.injectUser(user)
```

Comment:
Not a fan of this, this seems like it's essentially using an injected user as a toggle for the IndexRollupRequest to know whether or not it's coming from an external vs internal call. Why not use something like ConfigConstants.INTERNAL or something to be explicit since you're not actually using the injected user in any way. And if we plan to support injecting users then this needs to be removed in the future anyways.